src/app/pages/products-list/products-list.component.ts

```diff
@@ -14,11 +15,22 @@ export class ProductsListComponent implements OnInit {
 
     constructor(
         private readonly productsStoreService: ProductsStoreService,
+        private readonly activatedRoute: ActivatedRoute,
         private readonly router: Router,
     ) {}
 
     ngOnInit(): void {
-        this.productsStoreService.loadProducts();
+        this.activatedRoute.paramMap.pipe(
+            map(paramMap=>{
+                return paramMap.get("id")
+            }),
```

Comment:
Рекомендую в таких случаях сокращать запись
```suggestion
            map(paramMap => paramMap.get("id")),
```